src/Expense.java

```diff
@@ -1,7 +1,32 @@
+import java.util.Scanner;
 
 public class Expense {
+	Scanner scnr = new Scanner(System.in);

```

Comment:
Since a GUI will be used, it isn't feasible to require input from an IDE console. What would be needed is to set up constructors with parameters of each of the variables.  From the GUI, when the user presses the add button, input from each of the fields will be gathered, and can be used to create an object using an overloaded constructor.  

![image](https://github.com/Tbusk/Ewallet-Group2/assets/114008066/ae2404bf-c475-44f4-bdeb-b127a8feaed7)